src/main/scala/akka/persistence/jdbc/util/SlickDriver.scala

```diff
@@ -25,14 +25,24 @@ import slick.jdbc.JdbcProfile
 import slick.jdbc.JdbcBackend._
 
 object SlickDriver {
+
+  def forDriverName(config: Config): JdbcProfile =
```

Comment:
While I understand that it is good to have a backwards compatible version of this method. I think this api is actually an internal api (even though it is not marked as such).

Would it be an idea to mark this method as deprecated?

The same holds for the other overload you added below.

Reply:
I agree that this should be private. 

The problem now is that Lagom is using this API and because we use JNDI we don't have a path to pass.

Maybe we should add something similar in Lagom instead and not use the plugin one.

Reply:
Let's try the following:

* Lagom get's its own JNDI lookup utility class
* the old methods are reintroduced in this PR, but deprecated
* we make the new ones private/internal API

How does it sound?

Reply:
Sounds good